modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module CovidResearch
+  module Volunteer
+    class FormCryptoService
+      extend AttrEncrypted
+      attr_encrypted :submission, key: Settings.db_encryption_key
+
+      def decrypt_redis_format(redis_format)
+        decrypt_form(redis_format.form_data, redis_format.iv)
```

Comment:
Do we want to add any `Base64.decode`-ing after this? It seems a bit asymmetrical to have plain strings for input, but then output Base64-encoded strings.